### DISCUSSION TITLE
Is `not:(…)` selector supported in v0.4.1?

### DISCUSSION THREAD

**Discussion by HynekS** [tailwindlabs/tailwindcss#9355] [Help]
Just a simple question: Is `not:()` selector supported in `0.4.1`? Unfortunately, I am not able to upgrade yet (`twin.macro` compatibility issues).

I've tried to use `"a:not(h2 a, h3 a, h4 a, h5 a, h6 a)": { … }` in `tailwind.config` `extends`, but the result was broken prose CSS (silent fail, the build was finished, but `.prose` classes were not applied).

Or is my syntax wrong? Tried to pass it to `document.querySelectorAll(…)` and it worked as expected 🤔

**Comment by wongjn** [comment]
Just tried it and seemed to work fine – could it be some other configuration that's interfering? Perhaps compare the output CSS with out the selector and the output CSS with it, and see if that provides any clues?

**Reply by HynekS** [reply]
Thank you for reply. It's a little cumbersome to inspect the generated styles (`twin.macro` moves them to `<style>` tags), but it is definitely possible and I'll give it a try.
